docs/workforce-management-framework/topics/con-release-notes.adoc

```diff
@@ -5,19 +5,26 @@
 {PROJECT_NAME} connects a business back-office portal application with its fleet of mobilized employees via a mobile application.
 Both mobile and portal applications are fully customizable.
 
-== About {WFM-RC-Version} Release
-{PROJECT_NAME} {WFM-RC-Version} is a new produce was previously in tech preview. The Features are listed below.
+== 1.1.0 Release
```

Comment:
Love it! I can build generator to grab this data from jira "Release notes" field so it will minimize hasle of uploading this? WDYT?

Reply:
I think we can definitely look into this, since our JIRA issues are public as well :+1: